devkit/bootstrap-unix.sh

```diff
@@ -90,7 +99,7 @@ else
 
     # -P Prevent failure by allowing the script to use pip as a dependency source
     # -X Do not start minion service
-    ${sudo} sh install_salt.sh -P -X
+    ${sudo} sh install_salt.sh -P -X ${install_type}
```

Comment:
can we add in the following

chown -R $USER:$USER *

We can do something nice later